robot/demos/RosArduinoBlinkLed/RosArduinoBlinkLed.ino

```diff
@@ -0,0 +1,42 @@
+/*
+ * rosserial Subscriber Example
+ * Blinks an LED on callback
+ */
+
+/* Running the code
+ launch roscore
+ cmd: roscore
+ run rosserial client application that forwards your arduino messages to the rest of ROS
+ cmd: rosrun rosserial_python serial_node.py /dev/ttyUSB0 (or ACM0, whatever name of your arduino port is)
+ To observe the messages being published
+ cmd: rostopic pub toggle_led std_msgs/Empty --once
+*/
+
+#include <ros.h>
+#include <std_msgs/Empty.h>
+int BUILTIN_LED = 13;
```

Comment:
I think the keyword is actually LED_BUILTIN and it's predefined based on which arduino-type microcontroller you're using... i think it's defined in Arduino.h or something